src/core/library.c

```diff
@@ -1498,6 +1498,33 @@ QuicLibraryGetGlobalParam(
         Status = QUIC_STATUS_SUCCESS;
         break;
 
+    case QUIC_PARAM_GLOBAL_STATISTICS_V2_SIZES: {
+        static const uint32_t StatSizes[] = {
+            QUIC_STATISTICS_V2_SIZE_1,
+            QUIC_STATISTICS_V2_SIZE_2,
+            QUIC_STATISTICS_V2_SIZE_3,
+            QUIC_STATISTICS_V2_SIZE_4
+        };
+        const uint32_t NumSizes = sizeof(StatSizes) / sizeof(StatSizes[0]);
+        uint32_t MaxSizes = *BufferLength / sizeof(uint32_t);
+        if (MaxSizes == 0) {
+            *BufferLength = NumSizes * sizeof(uint32_t); // Indicate the max size.
+            Status = QUIC_STATUS_BUFFER_TOO_SMALL;
+            break;
+        }
+        if (Buffer == NULL) {
+            Status = QUIC_STATUS_INVALID_PARAMETER;
+            break;
+        }
+        uint32_t ToCopy = MaxSizes < NumSizes ? MaxSizes : NumSizes;
```

Comment:
Would be better to use `min` for clarity.

Reply:
It's really the same thing, and I don't feel it matters all that much. I'd prefer to leave as is and not make another change.

Reply:
IMO, it is worth the effort now - code gets written once and read many times, and `min` is straighforward when a ternary forces the reader to think to now if this is a min / max / something else.

One click option :)
```suggestion
        uint32_t ToCopy = CXPLAT_MIN(MaxSizes, NumSizes);
```